spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -85,7 +85,10 @@ public static void startMetastoreAndSpark() {
         CatalogUtil.loadCatalog(HiveCatalog.class.getName(), "hive", ImmutableMap.of(), hiveConf);
 
     try {
-      catalog.createNamespace(Namespace.of("default"));
+      Namespace defaultNamespace = Namespace.of("default");
```

Comment:
Do we need the try-catch after this change?

Reply:
In a perfect world no. 
However, this is just due diligence to avoid any unwanted failures.

Reply:
This is test code. I would remove anything which is not strictly needed.

Also getting an AlreadyExistsException after checking for existence should be a bug if there is no concurrency.

Reply:
Noted, removed try-catch while creating namespace in the latest commit :)